pkg/service/service_manager.go

```diff
@@ -40,6 +41,24 @@ func (sm *serviceManager) IsServiceExist(servicePath string) (bool, error) {
 	return true, nil
 }
 
+// IsServiceEnabled check if service is enabled
```

Comment:
nit: "check if service is enabled" -> "check if service is exists and enabled"

Reply:
done